Cargo.lock

```diff
@@ -0,0 +1,579 @@
+# This file is automatically @generated by Cargo.
```

Comment:
can we move the rust substreams example into a subpackage ?

Reply:
moved rust substreams setup to separate folder